udata_hydra/config.py

```diff
@@ -26,4 +26,7 @@
 # -- Webhook integration config -- #
 ENABLE_WEBHOOK = True
 UDATA_URI = 'http://dev.local:7000/api/2/'
-UDATA_URI_API_KEY = ''
+UDATA_URI_API_KEY = None
+
+# Save resource to minio or not
+SAVE_TO_MINIO = False
```

Comment:
Fix https://github.com/etalab/udata-hydra/issues/33

I've added a simple config now to activate/deactivate saving resource to minio